File: atcoder.jp/contests/aising2020/tasks/aising2020_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const odd = (n: number) => n % 2 === 1;

const main = () => {
  input();
  const A = input().split(" ").map(Number);

  const ans = A.filter((v, i) => odd(i + 1) && odd(v)).length;
  console.log(ans);
};

main();

File: atcoder.jp/contests/aising2020/tasks/aising2020_b/main.cpp
#include <bits/stdc++.h>
using namespace std;

bool odd(int n) { return n % 2 == 1; }

int main() {
  int N;
  cin >> N;

  vector<int> A(N);
  for (auto &&a : A) cin >> a;

  int ans = 0;
  for (int i = 0; i < N; i += 2) ans += odd(A[i]);

  cout << ans << endl;
}
